packages/mobile/src/geth/networkConfig.ts

```diff
@@ -73,6 +75,7 @@ const KOMENCI_LOAD_CHECK_ENDPOINT_PROD = 'https://mainnet-komenci.azurefd.net/v1
 
 const networkConfigs: { [testnet: string]: NetworkConfig } = {
   [Testnets.alfajores]: {
+    networkId: '44787',
```

Comment:
I realize we have a way to read this from the geth config, see https://github.com/celo-org/wallet/blob/6148646a29e5e719df9a1b32d1580ea7e6e61be8/packages/mobile/src/geth/geth.ts#L86
Would be nice to use/expose this instead (being careful to do it only after geth init has been done).

Reply:
Yeah, agreed. Will keep it in mind but not change for now. Those calls are async so we'd have to refactor to make getting the config a function.

Reply:
should we create an issue for this :)